app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationDetailsTypeHelper.scala

```diff
@@ -57,30 +57,33 @@ trait OrganisationDetailsTypeHelper extends Cacheable with I18nSupport {
   def registrationConnector: RegistrationConnector
   def appConfig: AppConfig
 
-  protected def handleOrganisationType(organisationType: OrganisationType)(implicit
+  protected def handleOrganisationType(
+    organisationType: OrganisationType,
+    businessVerificationCheck: Boolean = true
+  )(implicit
     request: JourneyRequest[AnyContent],
     executionContext: ExecutionContext,
     headerCarrier: HeaderCarrier
   ) =
     (organisationType.answer, request.isFeatureFlagEnabled(Features.isUkCompanyPrivateBeta)) match {
       case (Some(OrgType.UK_COMPANY), _) =>
-        getUkCompanyRedirectUrl()
+        getUkCompanyRedirectUrl(businessVerificationCheck)
           .map(journeyStartUrl => SeeOther(journeyStartUrl).addingToSession())
       case (Some(OrgType.OVERSEAS_COMPANY_UK_BRANCH), false) =>
-        getUkCompanyRedirectUrl()
+        getUkCompanyRedirectUrl(businessVerificationCheck)
           .map(journeyStartUrl => SeeOther(journeyStartUrl).addingToSession())
       case (Some(OrgType.SOLE_TRADER), false) =>
         getSoleTraderRedirectUrl()
           .map(journeyStartUrl => SeeOther(journeyStartUrl).addingToSession())
       case (Some(OrgType.REGISTERED_SOCIETY), false) =>
-        getRegisteredSocietyRedirectUrl()
+        getRegisteredSocietyRedirectUrl(businessVerificationCheck)
```

Comment:
REGISTERED_SOCIETY not mentioned on ticket - worth adding.

Reply:
Registered society is not listed as an option to be added in group organisation type.